src/sentry/static/sentry/app/plugins/jira/components/settings.tsx

```diff
@@ -59,9 +62,9 @@ class Settings extends DefaultSettings {
     });
   }
 
-  startEditing() {
+  startEditing = () => {
```

Comment:
Are these methods used as event handlers? They don't seem like they need to be bound functions.

Reply:
@markstory No, but we were binding `this` before and I wanted to be consistent.